### PR TITLE
fix: fixed the `siteName` not updating in tfm

### PR DESCRIPTION
**Pull request by IgnatG**
### Description
the `siteName` was not updating when eStore API calls were executing 

### Resolution
Updated the MongoDB calls to update the database differently

### CODE REVIEW

**Review by abhi-markan — DISMISSED** [review]

File: external-api/src/v1/controllers/estore/eStore.controller.ts [hunk 1]
```diff
@@ -98,25 +98,31 @@ export const createEstore = async (req: Request, res: Response) => {
...
           );
-          eStoreCronJobManager.start(siteCreationResponse.data.siteId);
+          eStoreCronJobManager.start(`Site${eStoreData.dealId}`);
         } else {
           console.error('API Call failed: Unable to create a new site in eStore %O', { siteCreationResponse });
           // update the database to indicate that the API call failed
-          await cronJobLogsCollection.updateOne({ dealId: { $eq: eStoreData.dealId } }, { $set: siteCreationResponse });
+          await cronJobLogsCollection.updateOne(
+            { dealId: { $eq: eStoreData.dealId } },
+            { $set: siteCreationResponse, 'siteCronJob.status': ESTORE_CRON_STATUS.FAILED, 'siteCronJob.failureDate': new Date() },
```
Comment (lmarrai-sw) [inline external-api/src/v1/controllers/estore/eStore.controller.ts:116]: Should this be
```suggestion
            { $set: { siteCreationResponse, 'siteCronJob.status': ESTORE_CRON_STATUS.FAILED, 'siteCronJob.failureDate': new Date() } },
```
?

File: external-api/src/cronJobs/eStoreTermStoreAndBuyerFolder.cron.ts [hunk 2]
```diff
@@ -67,18 +67,19 @@ export const eStoreTermStoreAndBuyerFolder = async (eStoreData: any) => {
     if (buyerFolderResponse?.status === 201) {
       console.info('API Call finished: The Buyer folder for %s was successfully created', eStoreData.buyerName);
 
+      const tfmDealsCollection = await getCollection('tfm-deals');
+      // update the `tfm-deals` collection once the buyer and deal folders have been created
```
Comment (lmarrai-sw) [inline external-api/src/cronJobs/eStoreTermStoreAndBuyerFolder.cron.ts:71]: Should the comment say
```suggestion
      // update the `tfm-deals` collection once the buyer folder has been created
```
instead?

**Review by lmarrai-sw — CHANGES_REQUESTED** [review]
Just two small comments from me:

File: external-api/src/v1/controllers/estore/eStore.controller.ts (hunk 1 above)
Comment (IgnatG) [reply external-api/src/v1/controllers/estore/eStore.controller.ts]: good spot! updated now

File: external-api/src/cronJobs/eStoreTermStoreAndBuyerFolder.cron.ts (hunk 2 above)
Comment (IgnatG) [reply external-api/src/cronJobs/eStoreTermStoreAndBuyerFolder.cron.ts]: good spot. I've updated that now

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**